jinahub/indexers/storage/PostgreSQLStorage/postgreshandler.py

```diff
@@ -128,6 +128,8 @@ def _create_table(self):
                 shard int,
                 last_updated timestamp with time zone default current_timestamp
             );
+            CREATE INDEX idx_shard ON {self.table}(shard);
```

Comment:
I think we talked about this and decided not to add an index as it would slow down? @maximilianwerk